### DISCUSSION TITLE
Meaning of the Percentage of Tests Failed

### DISCUSSION THREAD

**Discussion by DungeonMaestro215** [hneemann/Digital#911] [Q&A] [answered]
Hi all,

What is the meaning of the percentage of tests failed from the CLI testing output? I have a fairly simple combinational circuit that is currently failing 9 out of 16 rows on the Test truth table, but the CLI outputs "Tests: Failed (128%)." Does somebody know how this percentage is calculated? And if this is more specific to my example, then I can include further information.

Thanks

**Comment by hneemann** [comment, marked as answer]
There is simply a bug in the percentage calculation. ;-) See also #912 
Please try the latest [pre release](https://infdigital.dhbw-mosbach.de/).

**Reply by DungeonMaestro215** [reply]
Thank you! I will try it now.
